### DISCUSSION TITLE
Triage Users cannot assign Issues to Organization Projects

### DISCUSSION THREAD

**Discussion by askpatrickw** [community/community#21627] [Projects and Issues]
<p>We are using an Organization project.<br>
Repos are linked to the projects.<br>
Members all have Triage rights to all repos via a Team.<br>
That same Team has write access to the Project.</p>
<p>Users who have Triage rights, cannot assign issues to Projects which feels like triage to me.</p>
<p>Is this part of another permission or am I missing something completely?</p>

**Comment by canuckjacq** [comment]
<p>This is a known inconsistency I’m afraid. I’m adding your post to the internal tracking issue.</p>
<p>We recognise that the triage role is especially important to open source projects and so we are definitely interested in tailoring triage permissions in a way that makes real world sense.</p>
<p>I don’t have an ETA on this but I’ll do my best to update you when there is more to report.</p>

**Comment by askpatrickw** [comment]
<p>Thanks <a class="mention" href="/canuckjacq">@canuckjacq</a> for the update. Can you tell me which Role I should use to allow members to assign an issue to a project?</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>If you want them to be able to assign to a project in the issue sidebar, they would need repository write permissions.</p>
